sources/network/socks5.hpp

```diff
@@ -0,0 +1,444 @@
+#include <boost/asio.hpp>
```

Comment:
Can you add credit for `sehe` about `https://github.com/sehe/asio-socks45-client` in REDAME.md please in the section `#Reference` and add new subsection `#References - Librairies`

Reply:
Added a recognition of the author (`sehe`)